mypy/nodes.py

```diff
@@ -236,10 +238,10 @@ def __init__(self,
         else:
             self.ignored_lines = set()
 
-    def name(self) -> str:
+    def name(self) -> Bogus[str]:
```

Comment:
Hm, the above problem is probably only because of `MypyFile`. Can we just fix this instead by always setting `_name` in `__init__`. Also I suppose we can fix `_fullname` for `MypyFile` because the module `id` is always known immediately.